conductr_cli/terminal.py

```diff
@@ -1,6 +1,10 @@
 import subprocess
 
 
+def docker_info():
+    return subprocess.check_output(['docker', 'info']).strip()
```

Comment:
It appears to have been around for a while: https://docs.docker.com/v1.9/engine/reference/commandline/info/